src/new-client/src/App/LiveRates/Tile/Header/TileHeader.tsx

```diff
@@ -86,32 +85,27 @@ export const HeaderComponent: React.FC<{
   terms: any
   isTornOut?: boolean
   onClick?: () => void
-  tearOutMock?: boolean
+  supportsTearOut?: boolean
 }> = ({
   canTearOut,
   base,
   terms,
   date,
   isTornOut,
   onClick,
-  tearOutMock = false,
+  supportsTearOut = false,
 }) => {
   return (
-    <HeaderWrapper tearOutMock={tearOutMock}>
+    <HeaderWrapper supportsTearOut={supportsTearOut}>
       <TileSymbol data-qa="tile-header__tile-symbol">
         {base}/{terms}
       </TileSymbol>
       <DeliveryDate data-qa="tile-header__delivery-date">{date}</DeliveryDate>
-      {canTearOut && !tearOutMock && (
-        <HeaderAction onClick={onClick ? onClick : () => {}}>
+      {canTearOut && (
```

Comment:
This isn't visible in the styleguide for the hover state unless we hover over the tile. Sorry if I lead you down the wrong path in earlier comments.
I suggest adding an optional `hover` prop that sets the opacity so the styleguide can show it without manually hovering.